sortinghat/core/api.py

```diff
@@ -735,21 +735,54 @@ def merge_identities(from_uuid, to_uuid):
     The function raises a `NotFoundError` exception when either `from_uuid`
     or `to_uuid` do not exist in the registry.
 
-    :param from_uuid: identifier of the unique identity set to merge
-    :param to_uuid: identifier of the unique identity where `from_uuid`
+    :param from_uuids: List of identifiers of the unique identities set to merge
+    :param to_uuid: identifier of the unique identity where `from_uuids`
         will be merged
 
     :returns: a UniqueIdentity object as the result of the merged identities
 
-    :raises NotFoundError: raised when either `from_uuid` or `to_uuid`
-        do not exist in the registry
+    :raises NotFoundError: raised when either one of the `from_uuids` or
+        `to_uuid` do not exist in the registry
     """
-    def _merge_enrollments(trxl, from_uid, to_uid):
-        """Merge enrollments from two `UniqueIdentity` objects"""
-        # Get current enrollments from both uidentities
-        enrollments_db = from_uid.enrollments.all() | to_uid.enrollments.all()
-        enrollments = {}
 
+    def _find_unique_identities(from_uuids, to_uuid):
+        """Find the unique identities to be merged from their uuids"""
```

Comment:
Question: Should I include a black space between this _docstring_ line and the next one? I am not sure if this the right way to do it when it is an inner method.

Reply:
Yep. You should.

Reply:
Done, thank you!